guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+   image = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/silent-payments/silent-payments-header-mobile@2x.png"
+   alt-text = "."
+   width = 1600
+   height = 1000
+   layout = "full-width"
+%}
+
+# Silent payments
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+Bitcoin addresses are only meant to be used once. Thus, transaction flows usually start with users interacting with each other to specify the on-chain address(es) to be used in the transaction, with the goal of preventing on-chain address reuse. This not only adds steps to the user flow and adds friction by way of time and mental overhead on the part of users. BIP-352 specifies a protocol called Silent Payments that allows users to avoid such an interaction altogether, while ensuring on-chain addresses are not reused through a new type of static payment addresses.
+
+For example, Alice, who runs an NGO, can seek bitcoin donations from her supporters by posting a static address on her website, and receive donations at unique on-chain addresses without any other interaction with the donors.
+
+This is a significant improvement in user experience for on-chain transactions. Additionally, BIP-352 allows users to proactively add labels to their addresses that get recognised while payments are detected. This is a boost for coin control as well as the contacts features and can improve privacy.
+
+We will first look at Labels and Contacts, then delve into how they impact/improve other features.
+
+
+## Brief explanation
+
+A silent payment transaction happens in 4 broad steps:
+- The receiver shares/publishes a static payment address
+- The sender obtains & uses it to derive a unique on-chain address
+- The sender broadcasts a transaction that pays this derived address
+- The receiver identifies the payment as theirs by identifying that they control this address
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "."
+   width = 1024
+   height = 522
+   layout = "full-width"
+%}
+
+With this new model of transacting, BIP-352 introduces new primitives and concepts for users and developers. Below is a brief summary of them:
+
+| Current terminology        | With silent payments          | Benefit |
```

Comment:
I find it hard to decipher this table and would prefer full sentences.

Reply:
Updated in https://github.com/BitcoinDesign/Guide/pull/1109/commits/d9d953d4b4b4e0743ea38205a169406579e17ef6 hope this helps.

Reply:
The table has been updated for better phrasing but not replaced with full sentences since other s I've talked to didn't have this issue.